native/src/widget/scrollable.rs

```diff
@@ -162,9 +162,10 @@ pub fn layout<Renderer>(
     limits: &layout::Limits,
     width: Length,
     height: Length,
+    max_height: u32,
     layout_content: impl FnOnce(&Renderer, &layout::Limits) -> layout::Node,
 ) -> layout::Node {
-    let limits = limits.width(width).height(height);
+    let limits = limits.max_height(max_height).width(width).height(height);
```

Comment:
The output on 0.3.0 and on this branch look the same now :tada:

For `Scrollable`, I was surprised that `max_height()` gets superseded by `height(Length::Fill)` (far right column below), since that's not the case for `Container`, but maybe that's intentional and/or related to #715? It respects the `max_height()` if you set `height(Length::Shrink)` instead or just don't set a height.

Gist for screenshot: https://gist.github.com/mtkennerly/c67a18ffcef3b926926f012f7b6d1512

> <img width="770" alt="image" src="https://user-images.githubusercontent.com/4934192/179680320-6a586d5a-e3b1-4aed-bc4a-23e652bd27eb.png">

Reply:
Yeah. The layout engine overall needs a review and a proper test suite to ensure consistency.

I believe `max_` and `min_` attributes should eventually be moved to a single widget (maybe `Container`?).